File: DbModels/Review.cs
using BibliotecaOnline.Areas.Identity.DbModels;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace BibliotecaOnline.DbModels
{
    public class Review
    {
        [Key]
        public int ReviewId { get; set; }

        public double Rate { get; set; }
        public string Text { get; set; }
        public DateTime Data { get; set; }

        public int BookId { get; set; }
        public Book Book { get; set; }

        public int BibliotecaUserId { get; set; }
        public BibliotecaUser BibliotecaUser { get; set; }
    }
}

File: src/app/admin/models/review.ts
import { Book } from './book';
import { BibliotecaUser } from 'src/app/library/models/biblioteca-user';

export class Review {
    public reviewId: number;
    public rate: number;
    public text: string;
    public data: Date;
    public bookId: number;
    public book: Book;
    public bibliotecaUserId: number;
    public bibliotecaUser: BibliotecaUser;
}